app/main.py

```diff
@@ -16,4 +16,28 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+mutable = [list, dict, set]
+
+list_of_final_object_mut = []
+list_of_final_object_immut = []
+
+all_objects = [
+    lucky_number, pi, one_is_a_prime_number,
+    name, my_favourite_films, profile_info,
+    marks, collection_of_coins
+]
+
+for obj in all_objects:
+    we_go_to_loop = False
+    for obj_type in mutable:
+        if isinstance(obj, obj_type):
+            list_of_final_object_mut.append(obj)
+            we_go_to_loop = True
+            break
+    if we_go_to_loop is False:
+        list_of_final_object_immut.append(obj)
```

Comment:
No additional logic is required  in this task - simply pass the variables in the correct list in dict.